website/content/en/preview/getting-started-with-terraform/_index.md

```diff
@@ -217,11 +217,17 @@ resource "aws_iam_role_policy" "karpenter_controller" {
           "ec2:DescribeInstanceTypes",
           "ec2:DescribeInstanceTypeOfferings",
           "ec2:DescribeAvailabilityZones",
-          "ssm:GetParameter"
         ]
         Effect   = "Allow"
         Resource = "*"
       },
+      {
+        Action = [
+          "ssm:GetParameter"
+        ]
+        Effect   = "Allow"
+        Resource = "arn:aws:ssm:*:*:parameter/aws/service/*"
```

Comment:
Can we change this to `arn:*:ssm:*..` so it works across AWS partitions?

Reply:
ah yes, thats a good point. I changed that in the module but not here. There is a data source https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/partition we use that will pull the correct partition - I'll update to add this